runTests.ps1

```diff
@@ -87,9 +87,9 @@ Invoke-BuildStep 'Cleaning package cache' {
     -skip:(-not $CI) `
     -ev +BuildErrors
 
-Invoke-BuildStep 'Running /t:RestoreVS15' {
+Invoke-BuildStep 'Running /t:RestoreVS' {
```

Comment:
These changes were overlooked by https://github.com/NuGet/NuGet.Client/pull/2832.

Reply:
My bad, we don't use these scripts in our build at all, so I didn't even think about them.